test_scripts/Polices/build_options/ATF_P_Register_NewApp_not_exist_inLocalPT_FinishPTU_PROPRIETARY.lua

```diff
@@ -90,18 +92,29 @@ function Test:TestStep_RegisterNewApplication()
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = "Media Application" }})
   self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
   self.mobileSession2:ExpectNotification("OnPermissionsChange")
+  self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest", {requestType = "LOCK_SCREEN_ICON_URL"})
```

Comment:
@dboltovskyi, no thank you for notice. Done in 7411e599a53267114a219d4453f233242377f060